source/policies.ttl

```diff
@@ -0,0 +1,17 @@
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix : <https://id.kb.se/vocab/>
+base <https://id.kb.se/>
+
+</policy/freely-available> a :AccessPolicy ;
+    skos:prefLabel "Fritt tillgänglig"@sv, "Freely available"@en .
+
+</policy/gratis> a :AccessPolicy ;
+    skos:prefLabel "gratis"@sv, "gratis"@en ;
+    skos:altLabel "free"@en .
+
+</policy/restricted> a :UsageAndAccessPolicy ;
```

Comment:
Restricted on usage and/or access, or just on access?

Reply:
From https://en.unesco.org/open-access/sites/open-access/files/215863e.pdf

"The other variant, where material is free to read but does not explicitly permit further types of re-use, is called  ‘gratis’ Open Access." - So I think gratis and restricted both can be seen as the Access condition?

Reply:
I think these definitions need to be recorded in the term definitions, i.e. `description` and or `comment`

Reply:
#9923ae92a9abd6a705c67a641cdf11544c58d69b